### DISCUSSION TITLE
Does PYG support adversial attack on graphs?

### DISCUSSION THREAD

**Discussion by Tonniia** [pyg-team/pytorch_geometric#8508] [Ideas]
When searching using the keyword 'attack,' I observed that PYG only supports two methods(PRBCDAttack and GRBCDAttack) in the paper 'Robustness of Graph Neural Networks at Scale'. Surprisingly, I couldn't find support for the well-known Nettack (KDD-18 best paper). Moreover, I believe these valuable attack methods could be assimilated into perturbation-based explanations.

**Comment by rusty1s** [comment]
Hey, yes, currently we only support the two methods mentioned above. Sorry :( I am happy to include more but it might take some time.

**Comment by nowyouseemejoe** [comment]
Just found this repo: [GreatX](https://github.com/EdisonLeeeee/GreatX)
